tests/linearizability/model.go

```diff
@@ -46,11 +50,17 @@ type EtcdResponse struct {
 	Err          error
 }
 
+type EtcdLease struct {
+	LeaseID int64
+	Keys    map[string]bool
```

Comment:
What does the value (of type `bool`) mean?  If we don't care about the value at all, then I'd suggest to change it to  below,
```suggestion
	Keys    map[string]struct{}
```

Reply:
updated